CHANGELOG.md

```diff
@@ -41,7 +47,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Updated `GET /granules` endpoint to leverage ElasticSearch search-after API.
     The endpoint will only use search-after when the `searchContext` parameter
     is provided in a request.
-
+  
```

Comment:
Might need to configure your editor to remove white space on save.

Reply:
Updated the editor setting to trim trailing whitespace